app/mender.go

```diff
@@ -525,6 +525,7 @@ func spinEventLoop(c *ControlMapPool, to State, ctx *StateContext, controller Co
 		case "continue":
 			return to
 		case "pause":
+			log.Infof("Update Control: Pausing the update in %s state", mapState)
```

Comment:
The original message said "Update Control: Pausing before entering %s state", which is technically more correct, it pauses between the states, before state scripts and such, not in the state.

Reply:
I agree, fixing